.github/workflows/deploy-nextjs-app-prod.yaml

```diff
@@ -0,0 +1,39 @@
+name: Deploy website to prod
+
+on:
+  push:
+    branches: [main]
+    tags:
+      - 'v*'
```

Comment:
I assume this means it only deploys tagged v* tagged commits on main? I'd like to remove this and just deploy every commit to main.

Reply:
Roger that. And yes it means "deploy every tagged commit".